data-processing/common/parse_equation.py

```diff
@@ -53,9 +57,12 @@ def get_symbols(mathml: str) -> List[Symbol]:
     node_symbols = []
     consecutive_char_sequence: List[BeautifulSoup] = [] # The symbol_nodes of consecutive characters
     consecutive_char_indices: List[int] = [] # Indices of the characters of a consecutive character sequence
-    # It's 'safe' to call soup.find_all as the underlying bs4 implementation 
+    # It's 'safe' to call soup.find_all as the underlying bs4 implementation
     # appears to preserve order https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/bs4/element.py
     for symbol_node in soup.find_all(SYMBOL_TAGS):
+        # Skip partial symbols that aren't themselves descendants of another symbol
+        if symbol_node.name in PARTIAL_SYMBOLS and not symbol_node.parent.name in SYMBOL_TAGS:
+            continue
```

Comment:
I like the brevity, kudos! Isn't another way of implementing this just adding `mn` to the list of `CHARACTER_TAGS`, or am I missing something? One advantage of that approach is that characters are already another form of "partial symbols" (i.e. they're the pieces symbols are made up of), so then we only one data structure to hold 'em all.

Reply:
Ah I see hahah, I think I misunderstood what characters were and what exactly symbols were but it appears that yes that is all that is needed LOL

Reply:
Cool! My bad for my lackluster documentation.